client/src/cmdlfhid.c

```diff
@@ -475,26 +475,27 @@ static int CmdHIDClone(const char *Cmd) {
 static int CmdHIDBrute(const char *Cmd) {
     CLIParserContext *ctx;
     CLIParserInit(&ctx, "lf hid brute",
-                  "Enables bruteforce of HID readers with specified facility code.\n"
-                  "This is a attack against reader. if cardnumber is given, it starts with it and goes up / down one step\n"
-                  "if cardnumber is not given, it starts with 1 and goes up to 65535",
-                  "lf hid brute -w H10301 --fc 224\n"
-                  "lf hid brute -w H10301 --fc 21 -d 2000\n"
-                  "lf hid brute -v -w H10301 --fc 21 --cn 200 -d 2000\n"
-                  "lf hid brute -v -w H10301 --fc 21 --cn 200 -d 2000 --up\n"
+                  "Enables bruteforce of HID readers while maintaining specific parameter values.\n"
+                  "If the field being bruteforced is provided, it starts with it and goes up / down one step while maintaining other supplied values.\n"
+                  "If the field being bruteforced is not provided, it will iterate through the full range while maintaining other supplied values.",
+                  "lf hid brute -w H10301 -f fc --fc 224 --cn 6278\n"
+                  "lf hid brute -w H10301 -f cn --fc 21 -d 2000\n"
+                  "lf hid brute -v -w H10301 -f cn --fc 21 --cn 200 -d 2000\n"
+                  "lf hid brute -v -w H10301 -f fc --fc 21 --cn 200 -d 2000 --up\n"
                  );
 
     void *argtable[] = {
         arg_param_begin,
         arg_lit0("v", "verbose",             "verbose output"),
         arg_str1("w", "wiegand", "<format>", "see " _YELLOW_("`wiegand list`") " for available formats"),
-        arg_u64_0(NULL, "fc",     "<dec>",    "facility code"),
-        arg_u64_0(NULL, "cn",     "<dec>",    "card number to start with"),
-        arg_u64_0("i",  "issue",  "<dec>",    "issue level"),
-        arg_u64_0("o", "oem",     "<dec>",    "OEM code"),
-        arg_u64_0("d", "delay",   "<dec>",    "delay betweens attempts in ms. Default 1000ms"),
-        arg_lit0(NULL, "up",                 "direction to increment card number. (default is both directions)"),
-        arg_lit0(NULL, "down",               "direction to decrement card number. (default is both directions)"),
+	arg_str1("f", "field", "<fc, cn>",   "field to bruteforce"),
```

Comment:
param `f` is used for filename,   let me suggest not to use a short param name, replace with NULL and adapt help text accordingly

Reply:
Updated per suggestions. Also updated help text to accommodate.